logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -338,8 +350,72 @@ actual class CallManagerImpl(
         return AvsCallBackError.NONE.value
     }
 
+    /**
+     * onCallingReady
+     * Will start the handlers for: ParticipantsChanged, NetworkQuality, ClientsRequest and ActiveSpeaker
+     */
+    private fun onCallingReady() {
+        val participantChangedHandler = ParticipantChangedHandlerImpl(
+            onParticipantsChanged = { conversationId, participants, clients ->
+                onParticipantsChanged(
+                    conversationId = conversationId,
+                    participants = participants
+                )
+
+                scope.launch {
+                    withCalling {
+                        wcall_request_video_streams(
+                            inst = deferredHandle.await(),
+                            convId = conversationId,
+                            mode = DEFAULT_REQUEST_VIDEO_STREAMS_MODE,
+                            json = clients.toJsonString()
+                        )
+                    }
+                }
+            }
+        )
+
+        scope.launch {
+            withCalling {
+                wcall_set_participant_changed_handler(
+                    inst = deferredHandle.await(),
+                    wcall_participant_changed_h = participantChangedHandler,
+                    arg = null
+                )
+            }
+        }
+
+        // TODO: Network Quality handler
+        // TODO: Clients Request handler
+        // TODO: Active Speakers handler
+    }
+
+    private fun onParticipantsChanged(
+        conversationId: String,
+        participants: List<Participant>
+    ) {
+        val callProfile = _callProfile.value
+
+        callProfile[conversationId]?.let {
+            callingLogger.i("onParticipantsChanged() - conversationId: $conversationId")
+            participants.forEachIndexed { index, participant ->
+                callingLogger.i("onParticipantsChanged() - Participant[$index/${participants.size}]: ${participant.id}")
+            }
```

Comment:
This is just for test purposes, it hasn't been removed as more implementations will come.